test/transduce.js

```diff
@@ -40,6 +41,9 @@ describe('transduce', function() {
     eq(R.transduce(R.filter(isOdd), R.flip(R.append), [],  [1, 2, 3, 4]), [1, 3]);
     eq(R.transduce(R.compose(R.map(add(1)), R.take(2)), R.flip(R.append), [],  [1, 2, 3, 4]), [2, 3]);
     eq(R.transduce(R.compose(R.filter(isOdd), R.take(1)), R.flip(R.append), [],  [1, 2, 3, 4]), [1]);
+    eq(R.transduce(R.compose(R.uniq(), R.map(square), R.take(3)), R.flip(R.append), [],  [1, 1, 2, 2, 3, 3, 4, 4]), [1, 4, 9]);
```

Comment:
Was expecting `R.compose(R.uniq, …)` instead of `R.compose(R.uniq(), …)`. Not sure if that's an issue though but thought I would ask.

Reply:
Yes, that should be fixed.  Because of the way our curry works, `ramdaFunc()` yields back the equivalent to `ramdaFunc`, but we should not abuse that.

Reply:
Yes, there's no need to call with arity 0, I've removed the brackets.